packages/datadog-instrumentations/src/langchain.js

```diff
@@ -35,9 +28,10 @@ const extensions = ['js', 'cjs']
 
 for (const extension of extensions) {
   addHook({ name: '@langchain/core', file: `dist/runnables/base.${extension}`, versions: ['>=0.1'] }, exports => {
-    const RunnableSequence = exports.RunnableSequence
-    shimmer.wrap(RunnableSequence.prototype, 'invoke', invoke => wrapLangChainPromise(invoke, 'chain'))
-    shimmer.wrap(RunnableSequence.prototype, 'batch', batch => wrapLangChainPromise(batch, 'chain'))
+    if (extension === 'cjs') {
+      wrap(exports.RunnableSequence.prototype, 'invoke', 'orchestrion:@langchain/core:RunnableSequence_invoke')
```

Comment:
Any way to configure these names? It would be nice if we could update the publisher to the new approach without the subscriber having to know about it.

Reply:
The prefix of `orchestrion:${moduleName}:` is by design so we always know where this is coming from.

Not only that, the assumption that the subscriber wouldn't have to know about it is incorrect, because orchestrion doesn't do some of the processing previously done in the publisher, and uses different property names on the context object.